graveyard.json

```diff
@@ -1494,5 +1495,13 @@
     "link": "https://en.wikipedia.org/wiki/Field_Trip_(application)",
     "name": "Field Trip",
     "type": "app"
+  },
+  {
+    "dateClose": "2019-07-05",
+    "dateOpen": "2018-08-01",
+    "description": "The Dragonfly search engine was reportedly designed to link users' phone numbers to their search queries and censor websites such as Wikipedia and those that publish information about freedom of speech,[10] human rights, democracy, religion, and other issues considered sensitive by the Chinese government.",
+    "link": "https://en.wikipedia.org/wiki/Dragonfly_(search_engine)",
+    "name": "Dragonfly (Search Engine)",
```

Comment:
Just `Dragonfly` for the name. The `(search engine)` part is for Wikipedia's entry.